pvlib/pvsystem.py

```diff
@@ -831,7 +834,7 @@ def physicaliam(aoi, n=1.526, K=4., L=0.002):
 
```

Comment:
I agree with your suggestions, especially after having consulted my D&B book.  I would suggest calculating reflectance and absorption separately before combining them in order to clarify the code.